### ISSUE TITLE
Provide alternate image registry to dockerhub

### ISSUE THREAD

**Issue by onedr0p** [kubernetes/dashboard#7598]
### What would you like to be added?

Hello,

In order to combat dockerhub's imposed rate limits it would be great to host these images on ghcr or quay.io. Are there on any thoughts from the community on this initiation?

### Why is this needed?

Hello,

In order to combat dockerhub's imposed rate limits it would be great to host these images on ghcr or quay.io. Are there on any thoughts from the community on this initiation?

**Comment by jcpunk** [comment]
I'm also interested in a less rate limited repository as the default (or an option).

**Comment by shu-mutou** [comment]
Applying Docker-Sponsored Open Source Program may be a option.


https://www.docker.com/community/open-source/application/

**Comment by floreks** [comment]
I already did that. Waiting for reply.

**Comment by sftim** [comment]
/sig release
/sig k8s-infra

(AIUI)

**Comment by afbjorklund** [comment]
Looks like https://hub.docker.com/u/kubernetesui was accepted ?

**Comment by floreks** [comment]
Yep, we are part of their open source program now. We will still be looking into adding the ability to deploy to k8s org ghcr registry.

**Comment by cbrherms** [comment]
@floreks As above it looks like the images weren't marked public after the ci rework last year.
